docs/topics/scripting/custom-script-deps-tutorial.md

```diff
@@ -0,0 +1,332 @@
+[//]: # (title: Get started with Kotlin custom scripting – tutorial)
+
+> Kotlin scripting in [Experimental](components-stability.md). It may be dropped or changed at any time.
+> Use it only for evaluation purposes. We appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+>
+{type="warning"}
+
+_Kotlin scripting_ is the technology that enables executing Kotlin code as scripts, without prior compilation or
+packaging into executables.
+
+For an overview of Kotlin scripting with examples, see the talk [Implementing the Gradle Kotlin DSL](https://kotlinconf.com/2019/talks/video/2019/126701/)
+by Rodrigo Oliveira from KotlinConf'19 .
+
+In this tutorial, you'll create a Kotlin scripting project that executes arbitrary Kotlin code with Maven dependencies.
+You'll be able to execute scripts like the following:
+
+```kotlin
+@file:Repository("https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven")
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-html-jvm:0.7.3")
+
+import kotlinx.html.*; import kotlinx.html.stream.*; import kotlinx.html.attributes.*
```

Comment:
These imports are in a single line to reduce the LoC number?

Reply:
yes, but let's split them to follow the common conventions.